src/params.rs

```diff
@@ -358,7 +355,7 @@ fn parity(value: &[u8]) -> bool {
         v ^= v >> 4;
         v ^= v >> 2;
         v ^= v >> 1;
-        count += (v & 1) as i8
+        count += (v & 1) as i64
```

Comment:
same here

Reply:
should be ok since it's a byte being cast to i64